### ISSUE TITLE
The asc compiler doesn't work out of the box

### ISSUE THREAD

**Issue by ivanceras** [AssemblyScript/assemblyscript#2299]
I followed the instration from the README and there was no problem running the command lines
```
git clone https://github.com/AssemblyScript/assemblyscript.git
cd assemblyscript
npm install
npm link
```
I tested out with some simple examples:

fib.ts
```ts
export function fib(n: i32): i32 {
  var a = 0, b = 1
  if (n > 0) {
    while (--n) {
      let t = a + b
      a = b
      b = t
    }
    return b
  }
  return a
}

But compiling it gives me this error.
```shell
asc fib.ts --out fib.wasm --optimize
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '<home>/assemblyscript/dist/asc.js' imported from <home>assemblyscript/bin/asc.js
```

I investigated further and see that there is no `/dist/` directory inside of my `assemblyscript/` folder.
Did I miss some step or perhaps a wrong configuration in my npm/node installation?

**Comment by ivanceras** [comment]
```
node --version
v16.14.2
```

**Comment by ivanceras** [comment]
I just run: `npm run build` and I can that the `dist/` directory is populated with files and it works now.

**Comment by JairusSW** [comment]
Ah, okay! Do you still need this issue open?

**Comment by ivanceras** [comment]
Perhaps an additional `npm run build` instruction should be on the README for different node versions?

**Comment by MaxGraey** [comment]
It should be fixed in 0.20.7. Check this out
